code/online_challenges/src/project_euler/problem_018/problem_018.py

```diff
@@ -0,0 +1,11 @@
+def main():
+	prob = []
+	for i in range(15):
+		prob.append([int(x) for x in input().strip().split()])
```

Comment:
Instead of reading in from input, you can store the input in a data structure of choice.